packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -240,8 +248,13 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
     final TestWidgetInspectorService service = TestWidgetInspectorService();
     WidgetInspectorService.instance = service;
 
-    tearDown(() {
+    tearDown(() async {
       service.resetAllState();
+
+      await service.testBoolExtension(
```

Comment:
cc @goderbauer this was a real yak shave, but with this addition I was able to remove the no-shuffle annotation

Reply:
argh, maybe not, failing on mac